cmd/commands/subvolume.go

```diff
@@ -0,0 +1,41 @@
+package command
+
+import (
+	"github.com/rook/kubectl-rook-ceph/pkg/subvolume"
+	"github.com/spf13/cobra"
+)
+
+// SubvolumeCmd represents the rook command to manage subvolumes
+var SubvolumeCmd = &cobra.Command{
+	Use:   "stale-subvolume",
+	Short: "manage subvolumes",
+	Args:  cobra.ExactArgs(1),
+}
+
+var listCmd = &cobra.Command{
+	Use:                "ls",
+	Short:              "Print the list of stale subvolumes no longer in use.",
+	DisableFlagParsing: true,
+	Args:               cobra.MaximumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		clientsets := GetClientsets(cmd.Context())
+		VerifyOperatorPodIsRunning(cmd.Context(), clientsets, OperatorNamespace, CephClusterNamespace)
+		subvolume.List(cmd.Context(), clientsets, CephClusterNamespace, OperatorNamespace)
```

Comment:
```suggestion
		subvolume.List(cmd.Context(), clientsets, CephClusterNamespace, OperatorNamespace)
```
at L19 we are saying maximum 1 args but I don't see we are checking the args